### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by nilsbraden** [twhittock/avatto_me167#2]
I'm astonished, github has no messageing feature so I need to post it here...

I came across your repo while trying to add a converter for this "Cloud Even Smartes Heizkörper-Thermostat Zigbee Smart TRV LCD" (https://www.amazon.de/gp/product/B097MCYVDC) and since I failed I'm looking for another zigbee thermostat. This one let a lot of water run through after a while although the set temperature was way below room temperature so probably a hardware fault but I'm a bit tired of sending things back and forth.

So the device you linked in the issue #13727 at zigbee2mqtt seems to be in perfect order with your custom converter, right? Any trouble with the device warming up a room way too much? Had this with old danfoss zwave-devices... :/

Would love to hear about this device, maybe even put a note in the README.md explaining the current status of the converter! Thank you for your time uploading this here! ✌

**Comment by twhittock** [comment]
Hi, yeah it works.... But you should know these TRVs aren't very accurate - I'm not aware of any which don't require some sort of external calibration. It's not too surprising - the TRV is right next to the source of heat, and there's no way to know what the temperature is elsewhere in the space the radiator is meant to be heating, unless you tell it. At this moment, the me167 i use is set to a +5 C calibration offset. It was +3 C earlier today, so it needs constant adjustment. The same is true of the other TRV I use.

I'll add some info to the readme when i get some time.

**Comment by nilsbraden** [comment]
Thank you, that sounds good to me, just ordered 3 devices. :)

**Comment by nilsbraden** [comment]
Addendum:

I ordered three of the mentioned devices (the ones "linked in the issue #13727 at zigbee2mqtt") and they seem to work alright. The temperature offset is now set to +3°C and only with this they show a temperature which is acceptably close to the measured temperature right next to each device. I set this up for testing and found the thermostats to be at least 2.5 to 3.5°C off but the temperature-offset also changes the displayed temperature so it's usable like this.
